config.example/gpu-dashboard.json

```diff
@@ -1153,27 +1272,28 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "sum(rate(node_network_transmit_bytes_total{instance=\"$hostname:9100\",device=~\"en.*\"}[$__interval]))",
+          "expr": "sum(rate(node_network_transmit_bytes_total{instance=\"$hostname:9100\",device=~\"enp.*\"}[$__interval]))",
           "format": "time_series",
           "hide": false,
-          "interval": "",
-          "intervalFactor": 10,
+          "interval": ".5m",
+          "intervalFactor": 2,
           "legendFormat": "Transmit",
           "refId": "B"
         },
         {
-          "expr": "sum(rate(node_network_receive_bytes_total{instance=\"$hostname:9100\",device=~\"en.*\"}[$__interval]))",
+          "expr": "sum(rate(node_network_receive_bytes_total{instance=\"$hostname:9100\",device=~\"enp.*\"}[$__interval]))",
           "format": "time_series",
           "hide": false,
           "instant": false,
-          "interval": "",
-          "intervalFactor": 10,
+          "interval": ".5m",
```

Comment:
I needed to add this default interval. For some reason the default of '15s' was not displaying any information and it only shows up for this graph if intervals of `m` are used.